### DISCUSSION TITLE
Astor, revisa este hilo por favor.

### DISCUSSION THREAD

**Discussion by Gianibruno** [AstorArico72/Proyecto-Transversal-1#14] [General]
Astor, tenemos un problema con los chicos que cuando vos haces un push al GitHub modificas un tal Header de la plataforma que nos impide a nosotros Pullear y Pushear nuestros archivos desde Java con el NetBeans, por favor, fíjate cual puede ser el problema por que tenemos que crear todo un proyecto nuevo y vincularlo otra vez al repositorio para poder usarlo. Muchas gracias.

**Comment by AstorArico72** [comment]
Honestamente no tengo idea de qué es éso.
Normalmente borro los archivos que subo manualmente. ¿Estará ahí el problema?

**Reply by Gianibruno** [reply]
Puede ser, puede que el tema este en subir los archivos manualmente que se modifique algo
